README.md

```diff
@@ -26,6 +26,15 @@ If you'd like your tests to run automatically when they change, we recommend the
 ```bash
 find . -name "*_shpec.sh" | entr shpec
 ```
+### Test structure
+`Shpec` would be very familiar if you already know *BDD*, and test framework such as `rspec`, `jasmine`, `mocha`, *et caetera*
+
+The two main constructs are `describe/end` to regroup test, and `it/end` to describe a specific test.
+
+In the `it` block you will be the assertions, whose syntax is describe in the next chapter.
+
+Since you spec will be sourced as any shell script you can put in it any shell command. (in fact `describe`, `it`, `end`, *matchers* are defined has shell function)
```

Comment:
This is a good thing to note! Let me think if there's a better way to word this...